test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -367,39 +353,96 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 	wildwestVCClients, err := wildwestclientset.NewForConfig(apiExportVWCfg)
 	require.NoError(t, err)
 
-	t.Logf("verify that the export resource is retrievable")
+	t.Logf("verify that the exported resource is retrievable")
 	cowboysProjected, err := wildwestVCClients.WildwestV1alpha1().Cowboys("").List(logicalcluster.WithCluster(ctx, logicalcluster.Wildcard), metav1.ListOptions{})
 	require.NoError(t, err)
 	require.Equal(t, 1, len(cowboysProjected.Items))
 
-	t.Logf("Ensuring the appropriate core/v1 resources are available")
 	dynamicVWClusterClient, err := kcpdynamic.NewClusterDynamicClientForConfig(apiExportVWCfg)
 	require.NoError(t, err, "error creating dynamic cluster client for %q", apiExportVWCfg.Host)
 
-	grantedGVRs := []schema.GroupVersionResource{
+	sheriffsGVR := schema.GroupVersionResource{Version: "v1", Resource: "sheriffs", Group: "wild.wild.west"}
+
+	claimedGVRs := []schema.GroupVersionResource{
 		{Version: "v1", Resource: "configmaps"},
 		{Version: "v1", Resource: "secrets"},
 		{Version: "v1", Resource: "serviceaccounts"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterroles"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterrolebindings"},
+		sheriffsGVR,
 	}
 
-	for _, gvr := range grantedGVRs {
+	t.Logf("verify that we get empty lists for all claimed resources because the claims have not been accepted yet")
+	for _, gvr := range claimedGVRs {
 		t.Logf("Trying to wildcard list %q", gvr)
```

Comment:
nit: `"Trying to wildcard list %q in virtual apiexport apiserver for apiexport %q`, gvr, apiserver.Name